img.py

```diff
@@ -116,6 +207,8 @@ def update_curr_img_obj(self):
 			# null obj
 			self.currImgObj = img_obj(self, nullImg)
 

```

Comment:
NITPICKY: Maybe add some comments here to make sure people know about the different states when we are updating an image. Whether we are in the list of images, or handling when we are in a cropped mode, and if nothing else we just set it to a null image.